flickit-assessment-data/src/main/resources/liquibase/changelog-0.37.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.17.xsd">
+
+    <changeSet id="0.37-01" author="Maziyar Gerami">
+        <addColumn tableName="fak_kit_version">
+            <column name="status_version" type="bigint" valueComputed="
+            CASE
+                WHEN status = 0 THEN 0
+                WHEN status = 1 THEN 1
+                ELSE -id
+            END"/>
```

Comment:
**Suggestion:** Modify the `addColumn` changeSet to ensure that the `status_version` computation is consistent with the business logic defined in the application layer, particularly aligning with how `statusVersion` is generated in `KitVersionMapper`. [general, importance: 5]
```suggestion
            <column name="status_version" type="bigint" valueComputed="
            CASE
                WHEN status = 0 THEN 0
                WHEN status = 1 THEN 1000  // Adjusted to match application logic
                ELSE -id
            END"/>
```